packages/sdks/javascript/Makefile

```diff
@@ -0,0 +1,7 @@
+test:
+	@docker build --force-rm -f Dockerfile-dev -t devopness_sdk-javascript .
+	@docker run devopness_sdk-javascript npm run test
+
+build:
+	@docker build --force-rm -f Dockerfile-dev -t devopness_sdk-javascript .
+	@docker run -v sdk-javascript:/usr/app/ devopness_sdk-javascript npm run build-api-models
```

Comment:
`build` should only do one thing and do it well: `build`. We should not invoke `docker run` during the build stage.
If you want to provide a simple target for building the SDK, I suggest separating it into two targets. Something like `build-docker-image` and `build-sdk-js` might be useful, as `build-sdk-js` might invoke the image build, if needed:
```make
build-image: ## Build Docker image
    ... do something
   
build-sdk-js: build-image ## Run SDK build process
    ... do other things after invoking `build-image`
```

Reply:
@jfoliveira resolved